src/webgpu/api/validation/encoding/beginRenderPass.spec.ts

```diff
@@ -82,28 +82,28 @@ g.test('color_attachments,device_mismatch')
       ? t.getDeviceMismatchedRenderTexture()
       : t.getRenderTexture();
 
-    const encoder = t.createEncoder('non-pass');
-    const pass = encoder.encoder.beginRenderPass({
-      colorAttachments: [
-        {
-          view: view0Texture.createView(),
-          clearValue: { r: 1.0, g: 0.0, b: 0.0, a: 1.0 },
-          loadOp: 'clear',
-          storeOp: 'store',
-          resolveTarget: target0Texture.createView(),
-        },
-        {
-          view: view1Texture.createView(),
-          clearValue: { r: 1.0, g: 0.0, b: 0.0, a: 1.0 },
-          loadOp: 'clear',
-          storeOp: 'store',
-          resolveTarget: target1Texture.createView(),
-        },
-      ],
-    });
-    pass.end();
-
-    encoder.validateFinish(!mismatched);
+    t.expectValidationError(() => {
```

Comment:
expectValidationError should only contain one WebGPU call so we can ensure the error is coming from the correct one

Same comment in several places below